kivy/tests/test_uix_bubble.py

```diff
@@ -1,6 +1,17 @@
 import pytest
 
 
+@pytest.mark.parametrize('prop_name', (
+    '_fills_row_first',
+    '_fills_from_left_to_right',
+    '_fills_from_top_to_bottom',
+))
+def test_a_certain_properties_exist_in_any_of_the_super_classes(prop_name):
+    from kivy.uix.bubble import Bubble
+    super_classes = Bubble.mro()[1:]  # exclude Bubble itself
+    assert any(hasattr(klass, prop_name) for klass in super_classes)
```

Comment:
I'd probably test directly that GridLayout has these properties, rather than super classes.

Reply:
Thanks, I fixed. I thought the previous code is more robust against changes.

Reply:
It is perhaps more robust to changes in super class, but is also too clever :) and perhaps too flexible. So my worry was that there are more ways for it to go wrong than checking grid layout directly. After all the worry is mainly someone changing grid layout and forgetting about bubble?

Reply:
> After all the worry is mainly someone changing grid layout and forgetting about bubble?

Yes.

> So my worry was that there are more ways for it to go wrong than checking grid layout directly

Yes, there might be. Since I don't come up with any good idea than the current and the previous code, I'm ok with the current code.

Reply:
Actually putting `issubclass(Bubble, GridLayout)` in the unittests might remove my concern.